tenablesc/scanresult.go

```diff
@@ -102,6 +102,14 @@ func (c *Client) GetScanResult(id string) (*ScanResult, error) {
 	return &resp, nil
 }
 
+func (c *Client) DeleteScanResult(id string) error {
```

Comment:
I admit being startled by not having implemented deletion of scan results, but then recalled that I went through the whole dance of learning how to scrub data out of (sigh, ipv4/v6 only not agent) repositories and didn't care about manually expiring the unimported scans. 

This all makes sense, thanks for the addition!

Reply:
Thank you!